tests/acceptance/29_simulate_mode/files_promises.cf

```diff
@@ -0,0 +1,34 @@
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+bundle agent test
+{
+  commands:
+    # add --verbose here and look at the .actual log for debugging sub policy runs
+    "$(sys.cf_agent) -Kf $(this.promise_filename).sub --simulate=manifest > $(this.promise_filename).actual 2>&1"
+      contain => in_shell,
+      handle => "agent_run_complete",
+      comment => "Run sub policy in manifest mode and capture output to $(this.promise_filename).actual file.";
+    "$(G.sed)"
+      args => " \
+-e 's,$(sys.workdir),WORKDIR,' \
+-e 's/[0-9]*\.changes/PID.changes/' \
+-e 's/[0-9]\{4\}-[0-9]\{2\}-[0-9]\{2\}[[:blank:]][0-9]\{2\}:[0-9]\{2\}:[0-9]\{2\}[[:blank:]][-+][0-9]\{4\}/TIMESTAMP/' \
```

Comment:
This should have `(Access|Change|Modify)` in front so that it doesn't change other timestamps.

Reply:
Mmm, but if there are any timestamps in the output those will vary depending on the time the test is run right?

Not that it will come up very much other than `(Access|Change|Modify)` unless we sent current date into the contents of a file.